### ISSUE TITLE
Highlight scrollable compartment list

### ISSUE THREAD

**Issue by NXXR** [DLR-SC/ESID#295]
Based on feedback we should increase the suggestion that the expanded compartment list is scrollable as the added scroll bar may not be enough.

We could add a box shadow to the bottom of the scrollable container to suggest more content hiding at the bottom of the container like this:
<details><summary>Mockups (click to expand)</summary>

![grafik](https://github.com/DLR-SC/ESID/assets/25636783/3bbcd02f-76f0-45d8-8779-b7c60db8d326)
![grafik](https://github.com/DLR-SC/ESID/assets/25636783/3ce532d8-66ad-49aa-a55b-4683cddaf9ee)

</details> 

by adding `box-shadow: inset 0 -10px 10px -10px #000;`  to the container

**Comment by lenarothenhaeusler** [comment]
I'm not quite sure, if the shadow makes it more clear. What about adding little arrows in both directions of the scroll bar?

**Comment by shami-sneha** [comment]
We can add the scrollbar and see how it looks , only if you allow me to do that .......

**Comment by JonasGilg** [comment]
> We can add the scrollbar and see how it looks , only if you allow me to do that .......

Yes, you can try it out and send some screenshots. We can then iterate over the design.

**Comment by shami-sneha** [comment]
![scrolling 2](https://github.com/DLR-SC/ESID/assets/131749075/f982e5d5-5245-4c0f-baae-f810762aa562)
![shadow1](https://github.com/DLR-SC/ESID/assets/131749075/fee20a05-277d-4162-8622-a5c188fa300d)
Do you mean like this ?

**Comment by NXXR** [comment]
@JonasGilg meant more like this:
![grafik](https://github.com/DLR-SC/ESID/assets/25636783/b78f14df-87a6-472e-9fbb-63591015e16f)

or with scrollbar arrows and indent shadow:
![grafik](https://github.com/DLR-SC/ESID/assets/25636783/142e2a29-fe55-45d4-9d58-dd31af2933b1)

@lenarothenhaeusler

**Comment by NXXR** [comment]
obsolete due to shorter compartment list.
May be revisited if the list grows longer.
